Example/Example/EditorDemoController.swift

```diff
@@ -195,11 +195,15 @@ class EditorDemoController: UIViewController {
         } else {
             html = ""
         }
-        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(changeFont))
         editorView.setHTML(html)
         editorView.becomeFirstResponder()
     }
 
+    @objc func changeFont() {
+        editorView.richTextView.defaultFont = UIFont.preferredFont(forTextStyle: .callout)
```

Comment:
Thinking about this... It might feel counter-intuitive that setting the `defaultFont` also modifies the current content's font.
Maybe a method `setBaseFont()` or something similar would be more intuitive 🤔

What do you think?
No hard feelings though 👍

Reply:
We talked about this and decide to keep the current name for now and add more documentation about the uses of default font.